src/components/postList/Post.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { CommentList } from '../commentsList/CommentsList';
+import { User } from '../user/User';
+
+export const Post: FC<Posts> = ({
+  title,
+  body,
+  user,
+  comments,
+}) => (
```

Comment:
аналоигично комментарию выше
```suggestion
export const Post: FC<Props> = ({ post }) => (
```